rest_framework_json_api/renderers.py

```diff
@@ -396,7 +396,7 @@ def build_json_resource_obj(cls, fields, resource, resource_instance, resource_n
             resource_data.append(('links', {'self': resource[api_settings.URL_FIELD_NAME]}))
         return OrderedDict(resource_data)
 
-    def render_relationship_view(self, data, accepted_media_type=None, renderer_context=None):
+    def render_relationship_view(self, data, accepted_media_type=None, renderer_context={}):
```

Comment:
@santiavenda2 fair point, I guess this package needs to add better exception handling then since right off the bat if `renderer_context` is the default of `None`, the first line of code will fail. We're looking at other options though as this package lacks documentation and doesn't appear to be active anymore. The added complexity layer on top of DRF which is on top of Django just makes it very hard to deal with.